emulate/emulate_queue.py

```diff
@@ -163,21 +166,60 @@ def retrieve_next_animation() -> Code:
 		return code_from_priority_queue
 	return retrieve_random_animation()
 
+
 def reset_dmx_values():
 	number_of_values = 5 * 28 * 3  # 5 rows, 28 columns, 3 color values
 	serialized = ",".join("0" * number_of_values)
 	redis_db.set('DMXvalues', serialized)
 
 
+def start_udp_server():
+	UDPServerSocket = socket.socket(socket.AF_INET, type=socket.SOCK_DGRAM)
```

Comment:
nazwa zmiennej jest z dużej, nie powinno tak być. To jest zarezerwowane dla klas